src/main/java/org/springframework/integration/dsl/config/IntegrationFlowBeanPostProcessor.java

```diff
@@ -106,9 +108,20 @@ public void afterSingletonsInstantiated() {
 				multicaster.addApplicationListener(applicationListener);
 			}
 		}
+
+		this.beanFactory.getBeansOfType(IntegrationFlow.class);
```

Comment:
Eh... Solution isn't good. In case of `@RequestScope` we have:
```
Expected: an instance of org.springframework.beans.factory.BeanCreationNotAllowedException
     but: <java.lang.IllegalStateException: No Scope registered for scope name 'request'> is a java.lang.IllegalStateException
```
What isn't something we are expecting.

I guess we can get deal with `this.beanFactory.getBeanDefinitionNames()` and checking their scope the same way as in the `processStandardIntegrationFlow()`, but I still believe that we could reach `scope` solution for entire `IntegrationFlow`...